src/main/java/com/zollingpaper/backend/auth/config/AccessorArgumentResolver.java

```diff
@@ -0,0 +1,45 @@
+package com.zollingpaper.backend.auth.config;
+
+import com.zollingpaper.backend.auth.dto.Accessor;
+import com.zollingpaper.backend.auth.exception.AuthErrorCode;
+import com.zollingpaper.backend.auth.exception.AuthException;
+import com.zollingpaper.backend.auth.service.AuthService;
+import com.zollingpaper.backend.auth.util.JwtCookieConsumer;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+@Component
+public class AccessorArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private final AuthService authService;
+    private final JwtCookieConsumer jwtCookieConsumer;
+
+    public AccessorArgumentResolver(AuthService authService, JwtCookieConsumer jwtCookieConsumer) {
+        this.authService = authService;
+        this.jwtCookieConsumer = jwtCookieConsumer;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(Authenticated.class) &&
+                parameter.getParameterType().equals(Accessor.class);
```

Comment:
```suggestion
        boolean hasAuthAnnotation = parameter.hasParameterAnnotation(Authenticated.class);
        boolean isAccessorClass = Accessor.class.isAssignableFrom(parameter.getParameterType());

        return hasAuthAnnotation && isAccessorClass;
```

`parameter.getParameterType().equals(Accessor.class);` 같은 경우에는 정확히 Accessor.class와 같은지 확인하고,
`Accessor.class.isAssignableFrom(parameter.getParameterType());`를 사용하면 Accessor이거나 Accessor를 상속하거나 구현한 타입인지 확인합니다. 다형성을 고려해서 아래 방식을 사용하는 경우가 더 많다고 하니 참고해주세요!

Reply:
오 이러면 코드가 좀 더 보기 좋겠네요~
반영하겠습니다.